ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/toolAdmin/ToolAdminDisplayStrings.properties

```diff
@@ -0,0 +1,83 @@
+containerVolumesLabel = Container Volumes
+containerVolumesHostPathLabel = Host Path
+containerVolumesContainerPathLabel = Container Path
+containerImageFieldSetLabel = Container Image
+containerImageNameLabel = Name
+containerImageTagLabel = Tag
+containerImageURLLabel = URL
+containerVolumesFromLabel = Container Volumes From
+containerVolumesFromNameLabel = Name
+containerVolumesFromTagLabel = Tag
+containerVolumesFromURLLabel = URL
+containerVolumesFromNamePrefixLabel = Name Prefix
+containerVolumesFromReadyOnlyLabel = Read Only
+#ToolImplementation
+toolImplementationFieldSetLabel = Tool Implementation
+toolImplementationImplementorLabel = Implementor
+toolImplementationImplementorEmailLabel = Implementor Email
+testToolDataInputFilesLabel = Sample Input Files
+testToolDataInputFilesColumnLabel = File Name(s)
+testToolOutputFilesLabel = Sample Output Files
+testToolOutputFilesColumnLabel = File Name(s)
+#PublicAppWindow
+publicAppNameLabel = App Name
+publicAppIntegratorLabel = App Integrator
+publicAppIntegratorEmailLabel = App Integrator Email
+publicAppDisabledLabel = App Disabled?
+
+#ToolTip Messages
+toolAdminHelp = <p>A <b>tool</b> is the software program (binary or script) that is installed on a server and integrated into the backend of the Discovery Environment (DE) for use in DE apps.</p><br/><p>Use this form to fill in the details regarding the Docker image and container for the tool so it can be registered with the DE.</p><br/><p><b>NOTE:</b>  At this stage, a dockerfile should already have been created (go <a href="https://pods.iplantcollaborative.org/wiki/display/HDFDE/Creating+a+Dockerfile" target="_blank">here</a> for more information on creating a dockerfile).</p><br/><p>Also, the image and data-container for the tool should have already been built and tested (go <a href="https://pods.iplantcollaborative.org/wiki/display/csmgmt/DE+Tool+Docker+Image+registration+and+installation" target="_blank">here</a> for detailed instructions).</p>
+inputFilesHelp = The list of paths to test input files in iRODS
+outputFilesHelp = The list of paths to expected output files in iRODS
+containerDeviceHelp = Device information for the Docker container associated with the tool
+containerVolumeHelp = Volume information for the Docker container associated with the tool. Volumes are designated directories that exist outside the union file system to allow for persistent data and are accessible by the host.
+containerVolumesFromHelp = A list of Docker containers this tool`s container should import volumes from.
+entryPointHelp = The entrypoint defines what should be run when the image is run, and is ideal when a tool consists of a single main program.
+imageTagHelp = A tag is a label applied to a Docker image in a repository. tags are how various images in a repository are distinguished from each other.
+workingDirectoryHelp = The working directory or path for any instructions within the dockerfile.
```

Comment:
I haven't seen the `Warning` messages from the Swagger docs for the `POST /admin/tools` endpoint included in the UI yet.

I think the `Use caution with Entrypoint settings` and `Use caution with Volumes settings` should be displayed somewhere appropriate, but obvious, in the UI where these settings can be changed.